analytics/src/main/java/brocoeur/example/analytics/service/scheduler/PokerScheduledTask.java

```diff
@@ -34,7 +36,20 @@ public class PokerScheduledTask {
     @Scheduled(fixedRate = 30000)
     public void executeScheduledTask() {
         LOGGER.info("Poker scheduled task started at : {}", dateFormat.format(new Date()));
-        final List<ServiceRequestStatus> listServiceRequestStatus = serviceRequestStatusRepository.findAllByStrategyAndStatus(POKER_RANDOM.toString(), "TODO").collectList().block();
+        final List<ServiceRequestStatus> listServiceRequestStatus = serviceRequestStatusService.findAllServiceRequestStatusByStrategyAndStatus(POKER_RANDOM.toString(), TODO);
         LOGGER.info("List of poker request : {}", listServiceRequestStatus);
     }
+
```

Comment:
Add javadoc explaining the goal of this Scheduled task.

Reply:
Added